crates/swift-bridge-ir/src/bridged_type/bridgeable_result.rs

```diff
@@ -228,7 +237,7 @@ impl BuiltInResult {
                     err_swift_type = err_swift_type
                 ),
                 TypePosition::SharedStructField => todo!(),
-                TypePosition::SwiftCallsRustAsyncOnCompleteReturnTy => todo!(),
+                TypePosition::SwiftCallsRustAsyncOnCompleteReturnTy => format!("なにわ"),
```

Comment:
What's this?

Reply:
Sorry... I forgot to remove this. I usually print some meaningless word to debug `swift-bridge`.